accounts/tests/test_user.py

```diff
@@ -370,7 +370,7 @@ def test_user_delete_keep_sounds_using_web_form(self, submit_job):
         self.assertRedirects(resp, reverse('front-page'))
 
         # Check loaded page contains message about user deletion
-        self.assertIn(resp.content, 'Your user account will be deleted in a few moments')
+        self.assertContains(resp, 'Your user account will be deleted in a few moments', status_code=302)
```

Comment:
I think the two tests with the redirect were broken before.
Debugging the test locally I found:
```
> resp.content
''
```
which would render the previous test as 
```
> '' in 'Your user account will be deleted in a few moments'
True
```

Reply:
Any suggestion for a meaningful test @ffont ?

Reply:
Hmm I'm not sure I understand. This is a message that should be rendered in the html page after a user deletes her account. `resp.content` should have the contents of the redirected page. If this is not the case, maybe there is some extra change that makes following redirects not default? I guess if the tests were broken before, they'd be failing?

Reply:
looking at the source code, the message is added using the messages framework:
https://github.com/MTG/freesound/blob/master/accounts/views.py#L1410
and then we do a redirect to the home page

so at this time that the `resp` object is being checked, it's a 302 rediret, with no html content. I think @Bomme is correct that the test never actually worked as expected.

This test should be updated to 1) check that the resp is a redirect to the home page, 2) actually load the home page (or follow the redirect) and check that the string exists correctly.

I see that other cases of `self.assertIn` are like:

```py
self.assertIn('Incorrect password', resp.content)
```

and this test was 
```py
self.assertIn(resp.content, 'Your user account will be deleted in a few moments')
```

which has the parameters around the wrong way

Reply:
Thanks @alastair for explaining 😄 
I think I found a good way of fixing the test.